docs/api/vast-tracker.md

```diff
@@ -375,3 +382,160 @@ vastTracker.on('creativeView', () => {
   // impression tracking URLs have been called
 });
 ```
+
+### notUsed()
+Must be called if the ad was not and will not be played (e.g. it was prefetched for a particular
+ad break but was not chosen for playback). This allows ad servers to reuse an ad earlier
+than otherwise would be possible due to budget/frequency capping. This is a terminal event;
+no other tracking events should be sent when this is used. Player support is
+optional and if implemented is provided on a best effort basis as it is not technically
+possible to fire this event for every unused ad (e.g. when the player itself is terminated
+before playback). This is a terminal event; no other tracking events should be sent when this is used.
```

Comment:
you already mentioned this in line 389 :D

Reply:
Removed it